deploy/docker-compose.graph.yml

```diff
@@ -0,0 +1,41 @@
+version: '3.1'
+
+volumes:
+  postgres:
+    driver: local
+
+services:
+  graph-node:
+    image: graphprotocol/graph-node
+    restart: always
+    ports:
+      - '8000:8000'
+      - '8001:8001'
+      - '8020:8020'
+      - '8030:8030'
+      - '8040:8040'
+    env_file:
+      - .env
+    depends_on:
+      - postgres
+  postgres:
+    image: postgres
+    restart: always
+    ports:
+      - '5432:5432'
+    command: ["postgres", "-cshared_preload_libraries=pg_stat_statements"]
+    env_file:
+      - .env
+    volumes:
+      - postgres:/var/lib/postgresql/data
+  subgraphs:
+    build:
+      context: ../../subgraphs/
```

Comment:
Will this change after opening subgraphs?

Reply:
Yes, we will change this when subgraph repo goes public.